content/docs/csidriver/features/powerflex.md

```diff
@@ -1003,3 +1003,84 @@ If such a node is not available, the pods stay in Pending state. This means pods
 Without storage capacity tracking, pods get scheduled on a node satisfying the topology constraints. If the required capacity is not available, volume attachment to the pods fails, and pods remain in ContainerCreating state. Storage capacity tracking eliminates unnecessary scheduling of pods when there is insufficient capacity.
 
 The attribute `storageCapacity.enabled` in `values.yaml` can be used to enable/disable the feature during driver installation using helm. This is by default set to true. To configure how often the driver checks for changed capacity set `storageCapacity.pollInterval` attribute. In case of driver installed via operator, this interval can be configured in the sample file provided [here](https://github.com/dell/csm-operator/blob/main/samples/) by editing the `--capacity-poll-interval` argument present in the provisioner sidecar.
+
+## Multi Availability Zones
+CSI-PowerFlex driver version 2.13.0 and above supports Multi Availability Zones for Block. NFS is not supported at this time.
```

Comment:
Change "CSI-PowerFlex driver" to "PowerFlex CSI driver" as you reference it below.

Reply:
Fixed